### DISCUSSION TITLE
Support "swap" and "roll" windows in App Layouts like a tiling window manager

### DISCUSSION THREAD

**Discussion by bergtholdt** [rxhanson/RectanglePro-Community#227] [Ideas]
I would like to have keyboard shortcuts for rolling the windows or swapping two windows in App Layouts. This would make the App Layouts basically a Tiling Window Manager.

**Comment by rxhanson** [comment]
Thanks for the feedback. What exactly do you mean by rolling the windows?

**Reply by bergtholdt** [reply]
Hi, I meant rolling the order of the windows in the layout. Usually I have a main view, e.g. editor or terminal in large and centered, and a side document, e.g. a web browser with a reference document. My monitor is large so when I want to search something in the browser I want to have that in the main view instead of my editor. So I'd roll the windows, swapping the roles of the reference view and the main view. Until I found what I was looking for (new reference) and roll back to my editor. Single keyboard shortcut for rolling or swapping would thus be a cool feature for my workflow.

**Reply by rxhanson** [reply]
Got it, thanks. I've long thought about furthering the tiling aspects, and my plan is to dedicate a feature release to tiling functionality when I get the time for it.

**Reply by brothell** [reply]
I would love tiling management. All other app with tiling seems to be clunky or not user friendly (yabai, aerospace).  

With all features in Rectangle Pro it would be awesome toolkit.
